source/isaaclab/isaaclab/sim/utils.py

```diff
@@ -198,7 +201,7 @@ def wrapper(prim_path: str | Sdf.Path, *args, **kwargs):
             # if successful, do not look at children
             # this is based on the physics behavior that nested schemas are not allowed
             if not success:
-                all_prims += child_prim.GetChildren()
+                all_prims += child_prim.GetFilteredChildren(Usd.TraverseInstanceProxies())
```

Comment:
Hm apply_nested won't work on instanced proxies so parsing them here is unnecssary. users have to make the asset uninstanced before they can do this function.

Reply:
Merging in your PR instead of these changes